src/main/java/com/chessmates/controller/EntityController.groovy

```diff
@@ -22,12 +20,12 @@ class EntityController {
      * Return a list of all team players
      */
     @GetMapping(value = 'players')
-    List<Player> getPlayers() { entityService.getPlayers() }
+    List getPlayers() { entityService.getPlayers() }
```

Comment:
Why did you decide to drop the Generic here completely? Can't we have something more detailed than Object? We at least know that this is Json I think?